<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewRequestDialog</class>
 <widget class="QDialog" name="AddNewRequestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Requests Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineSearchQuery">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSearch">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>581</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Uni</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Department</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Course</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDep">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboCourses">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textComments">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>361</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAddUni">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>290</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Chart</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Schule</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Course</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textScuhleComments">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>361</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboSchuleCourses">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAddSchule">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>280</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Chart</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Sprache</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Institut</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Diploma</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Schedule</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboSprache">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboInstitut">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDiploma">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboSchedule">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushSearch_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Chart</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="labelStudentData">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
